src/progaudi.tarantool/NetworkStreamPhysicalConnection.cs

```diff
@@ -36,6 +36,9 @@ public void Dispose()
 
         public async Task Connect(ClientOptions options)
         {
+            if (! options.ConnectionOptions.Nodes.Any()) 
+                throw new ClientSetupException("There are no any configured tarantool nodes");
```

Comment:
Please, change message to "There are zero configured nodes, you should provide one" in both places.